components/css/mixins/_fonts.scss

```diff
@@ -0,0 +1,10 @@
+$font-arial:Arial,"Helvetica Neue",Helvetica,sans-serif;
+$font-archivo-narrow:"Archivo Narrow","Arial Narrow",Arial,sans-serif;
+$font-archivo-black:'archivo_blackregular',"Arial Black","Arial Bold",Gadget,sans-serif;
```

Comment:
There is a bit of a naming difference between Archivo Narrow and Archivo Black. The reason is that Archivo Narrow is currently using a font hosted on Google Fonts while Archivo Black is using a font hosted on the server. Lets go ahead and replace Archivo Black's variable value with:
`'Archivo Black',"Arial Black","Arial Bold",Gadget, sans-serif;`
We'll also start loading this font via Google Fonts instead.